dashboard/app/views/gallery_activities/_gallery_activities.html.haml

```diff
@@ -7,7 +7,7 @@
   %div.gallery_activity
     - activity = gallery_activity.activity
     - level_source = activity.level_source
-    = link_to image_tag(original_image_url(level_source)), level_source
+    = link_to image_tag(level_source.level_source_image.s3_url), level_source
```

Comment:
Some templates have checks like "if level_source && level_source.level_source_image" but this one doesn't.  Does it need it?